media/css/core.css

```diff
@@ -1168,6 +1168,22 @@ div.module.project-subprojects div.subproject-meta {
     font-style: italic;
 }
 
+/* project delete page */
```

Comment:
Is really necessary to add more css classes? Can't we reuse the ones for a normal button?

Reply:
Yes, you are right. I will try to reuse as much CSS as I can.

Reply:
Hi @stsewd, I tried reusing the CSS classes for a normal button, it seems like the normal button text is smaller and doesn't look similar to the Delete forever button. I have to override ``display: block`` using ``display: inline !important;``, so that both buttons are inline.